src/types/describe/formatters/serialize.ts

```diff
@@ -35,7 +35,7 @@ const delegate: ReporterDelegate<Buffer, string, void> = {
   },
 
   emitNamedType({ type: { label }, buffer }): void {
-    buffer.push(`${label.name}`);
+    buffer.push(`"${label.name}"`);
```

Comment:
we should probably use `JSON.stringify(label.name)`, since `label.name` could in theory contain a slash